test/unistd/links.c

```diff
@@ -31,25 +31,66 @@ void setup() {
   rtn = write(fd, "test", 5);
   assert(rtn == 5);
   close(fd);
-  rtn = mkdir("folder", 0777);
+  rtn = mkdir("directory", 0777);
```

Comment:
Make we can use `makedir(name)` and `makefile(name, content)` helpers to make this setup more readable?

Reply:
(also makelink so that we can add the `assert(rtn == 0)` for those symlink calls)

Reply:
Should I replace every `mkdir` function for a `makedir` one, `write` function for a `makefile` and a `makelink` for any `symlink` function ?

Reply:
Yes, that is what I was thinking.

Then it would read something like:
```
makedir("test"):
makefile("test/file", "content");
makelink("file1", "file2");
```